### ISSUE TITLE
ugly crash when no subcommand specified

### ISSUE THREAD

**Issue by wbolster** [requires/api#3]
after a `pip install` i get this:

```
$ requires.io
Traceback (most recent call last):
  File "/home/wbolster/some-project/.direnv/python-3.6.2/bin/requires.io", line 11, in <module>
    sys.exit(main())
  File "/home/wbolster/some-project/.direnv/python-3.6.2/lib/python3.6/site-packages/requires_io/commands.py", line 235, in main
    return Commands().execute(args[1:])
  File "/home/wbolster/some-project/.direnv/python-3.6.2/lib/python3.6/site-packages/requires_io/commands.py", line 152, in execute
    return args.execute(args)
AttributeError: 'Namespace' object has no attribute 'execute'
```

**Comment by omansion** [comment]
thanks for the feeback, fixed in version 0.2.6

**Comment by wbolster** [comment]
awesome @omansion that was quick!
